.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+repos:
+- repo: git://github.com/antonbabenko/pre-commit-terraform
+  rev: v1.27.0
+  hooks:
+    - id: terraform_fmt
```

Comment:
I can recommend adding 
```
  - repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v2.0.0
    hooks:
      - id: trailing-whitespace
      - id: end-of-file-fixer
  - repo: https://github.com/awslabs/git-secrets
    rev: master
    hooks:
      - id: git-secrets
```
to your pre-commit hooks. the first ones for fixing the end of file in particular

Reply:
Done